Assets/00_Content/Scripts/Interactables/PickUp.cs

```diff
@@ -9,14 +9,22 @@ namespace Interactables {
 	public class PickUp : MonoBehaviour {
 
 		[SerializeField] private Transform itemGrabTransform;
-		[SerializeField] private Collider objectCollider;
 		[SerializeField] private LayerMask itemSlotLayer;
 
+		private Collider coll;
+		private MeshFilter meshFilter;
+
+		public void Start() {
+			coll = GetComponentInChildren<Collider>();
```

Comment:
I'm thinking that this might get the wrong collider.
For example, if you have a pickup with two colliders, one for collision and one for pickup detection. Then you might get the wrong one. Right?

Reply:
Yes that would be an issue, if we were to use more than one. But yeah, I could just make it a serialized field.